stratum/hal/lib/tdi/tdi_sde_interface.h

```diff
@@ -397,7 +402,7 @@ class TdiSdeInterface {
       int device, std::shared_ptr<TdiSdeInterface::SessionInterface> session,
       uint32 table_id, absl::optional<uint32> register_index,
       std::vector<uint32>* register_indices,
-      std::vector<uint64>* register_values, absl::Duration timeout) = 0;
+      std::vector<uint64>* register_datas, absl::Duration timeout) = 0;
```

Comment:
Tsk! You undid one of my grammar corrections. `Data` is plural (the singular is `datum`). I substituted `values`, which is a proper plural (and doesn't conflict with any other identifiers in this module).

This is one of the files I refactored. I can redo the correction when I merge the refactoring.

Reply:
Fixed